--- image_gallery.module	(revisão 734)
+++ image_gallery.module	(cópia de trabalho)
@@ -435,7 +435,7 @@
   if ($image->sticky) {
     $content .= ' class="sticky"';
   }
-  $content .= " style='height : {$height}px; width : {$width}px;'>\n";
+  //$content .= " style='height : {$height}px; width : {$width}px;'>\n";
   $content .= l(image_display($image, IMAGE_THUMBNAIL), 'node/'. $image->nid, array('html' => TRUE));
   $content .= '<h3>'. l($image->title, 'node/'. $image->nid) .'</h3>';
   if (variable_get('image_gallery_node_info', 0)) {
Index: gallery_navigation/gallery_navigation.module

